src/lib/rnp2_internal.h

```diff
@@ -0,0 +1,105 @@
+/*-
```

Comment:
I would prefer we drop this file and any other rnp2 reorganizing changes. It's not really neccessary in this PR and it's already going to be enough work to rebase.

(Plus I've already done some work towards a larger reorganization of the FFI.)

Reply:
Agreed. I've forgotten to do it